### PR TITLE
Eda

### LINKED ISSUE #10: EDA

### PR DESCRIPTION
**Pull request by Dana-Ismail**

### CODE REVIEW

File: clustering.ipynb [hunk 1]
```diff
@@ -64,6 +64,8 @@
     "import pandas as pd\n",
```
Comment (Reema-Khaseeb) [inline clustering.ipynb:64]: <pre class="ql-syntax" spellcheck="false">Line #3.    for key,value in obj.iteritems():
</pre><p>obj.iteritems() result a tuple of (key,value)</p><p>You could print out the resulted value in each iteration as one value, as it results the same : (a comma separated (key, value) in each line)</p><p>You could do:</p><pre class="ql-syntax" spellcheck="false">for key_value in obj.iteritems():
  print(key_value)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Reema-Khaseeb/DM-Assignment1-NYC-transportation/pull/14/discussion/#comment-1111219277'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (Reema-Khaseeb) [inline clustering.ipynb:64]: <p>Can you please clarify <strong>in code implementation</strong> how did you infer that (data['is_participant'] = 0) doesn't help?</p><p>To show the flow and reasons behind dropping these rows, e.g., the rows for most of the features are NULLs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Reema-Khaseeb/DM-Assignment1-NYC-transportation/pull/14/discussion/#comment-1111234695'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (Reema-Khaseeb) [inline clustering.ipynb:64]: <p>No correlation resulted!!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Reema-Khaseeb/DM-Assignment1-NYC-transportation/pull/14/discussion/#comment-1111234698'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (Reema-Khaseeb) [inline clustering.ipynb:64]: <p>What is the resulted intuition from this plot? </p><p>There may be another plot to be used for these categorical variables that result better!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Reema-Khaseeb/DM-Assignment1-NYC-transportation/pull/14/discussion/#comment-1111234699'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (Dana-Ismail) [reply clustering.ipynb:64]: done

Comment (Dana-Ismail) [reply clustering.ipynb:64]: the column will be dropped anyways

Comment (Dana-Ismail) [reply clustering.ipynb:64]: bar plots are the best way to visualize categorical data based on research.

Comment (Reema-Khaseeb) [reply clustering.ipynb:64]: <p>This is not a barplot</p>

**Review by mohamadhase — APPROVED** [review]

**Pull request merged**